ext/opentelemetry-ext-pymysql/src/opentelemetry/ext/pymysql/__init__.py

```diff
@@ -45,24 +45,30 @@
 
 import pymysql
 
-from opentelemetry.ext.dbapi import wrap_connect
+from opentelemetry.auto_instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.ext.dbapi import unwrap_connect, wrap_connect
 from opentelemetry.ext.pymysql.version import __version__
 from opentelemetry.trace import TracerProvider, get_tracer
 
 
-def trace_integration(tracer_provider: typing.Optional[TracerProvider] = None):
```

Comment:
Update the docstring to use `PymysqlInstrumentor.Instrument()`?

Reply:
Done.